src/AWS/Signers/V4.elm

```diff
@@ -1,4 +1,4 @@
-module AWS.Signers.V4 exposing (sign)
+module AWS.Signers.V4 exposing (sign, authorization)
```

Comment:
This doesn't necessarily have to be exposed. I just did it for ease of testing. It does bring up the question of how we test non-exposed functions if the runner has to import modules for testing.